src/modules/core/components/MembersList/MembersListItem.css

```diff
@@ -1,6 +1,9 @@
+@value reputationColor: rgb(255, 176, 0);
+
 .main {
```

Comment:
Please add hover state to the list, the same color as the one in the `ActionsList`, as well as a pointer cursor.

Clicking an item should bring up the "Edit Permissions" modal _(but you don't have to wire this up)_